types/stylelint/index.d.ts

```diff
@@ -168,9 +168,11 @@ declare namespace stylelint {
 		config?: Config;
 	};
 
+	type StylelintType = 'deprecation' | 'invalidOption' | 'parseError';
```

Comment:
[suggest] Since `StylelintType` is too generic, I suggest:

```suggestion
	type StylelintWarningType = 'deprecation' | 'invalidOption' | 'parseError';
```